controller/controller.go

```diff
@@ -764,6 +764,8 @@ func saveInstallResults(rootDir string, md *model.SystemInstall) error {
 		if len(errMsgs) > 0 {
 			return errors.Errorf("%s", strings.Join(errMsgs, ";"))
 		}
+	} else if md.IsTelemetryEnabled() && md.IsTelemetryInstalled() == false {
```

Comment:
Then this becomes the else half of the IsTelemeteryInstalled
```suggestion
	} else  {
```